packages/eslint-plugin-gestalt/src/button-icon-restrictions.js

```diff
@@ -50,7 +49,7 @@ const rule = {
           return;
         }
         importedComponent = decl.specifiers.some((node) => {
-          const isValidComponent = node.imported.name === componentName;
+          const isValidComponent = node.imported?.name === componentName;
```

Comment:
Discovered an issue with a prop called `name` on the `Example.js` page:

https://github.com/pinterest/gestalt/blob/fe1aa734f729e9ce5b249b0b820c7d466a180e18/docs/src/components/Example.js#L40